scripts/dev.js

```diff
@@ -45,7 +45,10 @@ const dev = async (maybeInit) => {
     await Promise.all([removeArtifacts()])
   }
   // Enable this if you're creating new github schemas
-  // await generate(codegenSchema)
+  try {
+    await generate(codegenSchema)
```

Comment:
note: need to figure out how to get eslint to ignore fixing up the autogenerated files before we uncomment this out for good